tests/unit/serialization/serialization_array.cpp

```diff
@@ -199,6 +199,35 @@ void test_plain_array()
     }
 }
 
+template <typename T, std::size_t SIZE>
+void test_array_of_vectors()
```

Comment:
Please don't use all-caps names (except single letter template parameters) - those are reserved for macros.